ReactApplication/src/Components/style.css

```
@@ -0,0 +1,153 @@
+.side-panel {
+    background-color:#f3d7ca;
+    width: 280px;
+    height: 100vh;
+    padding: 18px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+    align-items: center;
+    position: sticky;
+    top: 0%;
+}
+
+.cus-style{
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-radius: 2em;
+    font-weight: 600;
+    font-size: 16px;
+    margin-left: 10px;
+    margin-right: 10px;
+}
+
+.input-label-color{
+    width: 13.8em;
+    height: 2.2em;
+    background-color: #fff8e3;
+    margin-top: 1em;
+}
+.input-label{
+    width: 30px;
+    background-color: #fff8e3;
+    height: 2.2em;
+    width: 2.2em;
+    border-radius: 30px;
+    padding: 8px;
+    margin-top: 20px;
+}
+
+.flex-1{
+    display: flex;
+    flex-direction: row;
+    flex: 1;
+}
+
+
+.colorpicker{
+    margin-top: 20px;
+    margin-left: 10px;
+    margin-right: 10px;
+}
+
+.cus-btn{
+    width: 13.8em;
+    height: 2.2em;
+    background-color: black;
+    color: white;
+    margin-top: 20px;
+    font-weight: 400;
+}
+
+.cus-btn:hover{
+    background-color: #d0aa97;
+    color: black;
+    border: none;
+    font-weight: 600;
+    cursor: pointer;
+}
+
+
+.cus-input{
+    width: 13em;
+    height: 2.5em;
+    background-color: #fff8e3;
+    border-radius: 30px;
+    margin-top: 20px;
+    padding: 10px;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  max-width: 100%;
+  position: relative;
+}
+
+.heading {
+  display: flex;
+  background-color: #f3d7ca;
+  height: 3.6em;
+  align-items: center;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
+
+.heading h2 {
+  margin: 1.5em;
+  font-weight: 600;
+  color: rgb(47, 47, 47);
+}
+
+
+.grid-container{

```

```suggestion
.heading h2 {
  margin: 1.5em;
  font-weight: 600;
  color: rgb(47, 47, 47);
}


.grid-container {

```

do make sure to stay consistent for formatting.